src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/PublishingServiceTests.cs

```diff
@@ -407,23 +408,30 @@ public async Task PublishStagedReleaseContent()
         }
 
         private static PublishingService BuildPublishingService(
-            string? publicStorageConnectionString = null,
-            IBlobStorageService? privateBlobStorageService = null,
-            IBlobStorageService? publicBlobStorageService = null,
+            IPrivateBlobStorageService? privateBlobStorageService = null,
+            IPublicBlobStorageService? publicBlobStorageService = null,
             IMethodologyService? methodologyService = null,
             IPublicationRepository? publicationRepository = null,
             IReleaseService? releaseService = null,
-            ILogger<PublishingService>? logger = null)
+            ILogger<PublishingService>? logger = null,
+            IConfiguration? configuration = null)
         {
             return new PublishingService(
-                publicStorageConnectionString ?? "",
-                privateBlobStorageService ?? Mock.Of<IBlobStorageService>(MockBehavior.Strict),
-                publicBlobStorageService ?? Mock.Of<IBlobStorageService>(MockBehavior.Strict),
+                privateBlobStorageService ?? Mock.Of<IPrivateBlobStorageService>(MockBehavior.Strict),
+                publicBlobStorageService ?? Mock.Of<IPublicBlobStorageService>(MockBehavior.Strict),
                 methodologyService ?? Mock.Of<IMethodologyService>(MockBehavior.Strict),
                 publicationRepository ?? Mock.Of<IPublicationRepository>(MockBehavior.Strict),
                 releaseService ?? Mock.Of<IReleaseService>(MockBehavior.Strict),
-                logger ?? Mock.Of<ILogger<PublishingService>>()
+                logger ?? Mock.Of<ILogger<PublishingService>>(),
+                configuration ?? DefaultConfigurationMock().Object
             );
         }
+
+        private static Mock<IConfiguration> DefaultConfigurationMock()
+        {
+            return MockUtils.CreateMockConfiguration(
```

Comment:
Wondering what this is being used for now?

Reply:
If I switch out `DefaultConfigurationMock` for `Mock.Of<IConfiguration>()` in `BuildPublishingService`, I get several test failures on the `_publicStorageConnectionString = configuration.GetValue<string>("PublicStorage");` line in the `PublishingService` constructor. So I think it is still necessary.

Reply:
No worries, cheers for checking it out